src/Components/Performances.js

```diff
@@ -46,7 +46,7 @@ class Performances extends Component {
         <div className="row">
           <div className="three columns header-col">
             <h1>
-              <span>Past</span>
+              <span>Past Events</span>
```

Comment:
With this change, it looks asymmetrical compared to the "Upcoming". I suggest we leave it as is.